### ISSUE TITLE
Folder picker for picking a default folder path

### ISSUE THREAD

**Issue by Cephel** [mpv-android/mpv-android#174]
The current way to select a default start path is completely unusable if you're not very close to the root of the file hierarchy.

**Comment by jerryn70** [comment]
Yes it's true. Everytime i need to go through the files picker to choose videos from different folders.
I have few suggestions.
1) The app should scan for video files when opening/starting the app and after the scan it should display video files or  folder that contains a video file. 
The scan must run in background. Display a small scanning inductor to know the scan is running or not. 
Add an option to exclude particular folders from scan.
